inference-engine/tests/unit/cpu/mkldnn_memory_desc_test.cpp

```diff
@@ -45,9 +45,1126 @@ TEST(MemDescTest, Conversion) {
+        mkldnnMemDesc.redefineDesc(desc_ptr);
+        ASSERT_TRUE(mkldnnMemDesc.getDesc().isCompatible(*desc_ptr));
+        ASSERT_FALSE(mkldnnMemDesc.getDesc().isCompatible(*dnnl_blk_desc_orig));
+
+        // DnnlBlockedMemoryDesc
+        MKLDNNMemory mkldnnMemBlkDesc(eng);
+        mkldnnMemBlkDesc.Create(dnnl_blk_desc_orig);
+        DnnlBlockedMemoryDescPtr dnnl_blk_desc_ptr;
+        dnnl_blk_desc_ptr = std::make_shared<DnnlBlockedMemoryDesc>(shape, mkldnn::memory::data_type::u8, dnnl::memory::format_tag::nhwc);
+        mkldnnMemBlkDesc.redefineDesc(dnnl_blk_desc_ptr);
+        ASSERT_TRUE(mkldnnMemBlkDesc.getDesc().isCompatible(*dnnl_blk_desc_ptr));
+        ASSERT_FALSE(mkldnnMemBlkDesc.getDesc().isCompatible(*dnnl_blk_desc_orig));
+
+        // CpuBlockedMemoryDesc
+        MKLDNNMemory mkldnnMemCpuBlkDesc(eng);
+        mkldnnMemCpuBlkDesc.Create(dnnl_blk_desc_orig);
+        CpuBlockedMemoryDescPtr cpu_blk_desc_ptr = std::make_shared<CpuBlockedMemoryDesc>(dnnl_blk_desc_ptr->getPrecision(), dnnl_blk_desc_ptr->getShape(),
+                                                                                          dnnl_blk_desc_ptr->getBlockDims(), dnnl_blk_desc_ptr->getOrder(),
+                                                                                          dnnl_blk_desc_ptr->getOffsetPadding(),
+                                                                                          dnnl_blk_desc_ptr->getOffsetPaddingToData(),
+                                                                                          dnnl_blk_desc_ptr->getStrides());
+        mkldnnMemCpuBlkDesc.redefineDesc(cpu_blk_desc_ptr);
+        ASSERT_TRUE(mkldnnMemCpuBlkDesc.getDesc().isCompatible(*cpu_blk_desc_ptr));
+        ASSERT_FALSE(mkldnnMemCpuBlkDesc.getDesc().isCompatible(*dnnl_blk_desc_orig));
+    };
+
+    std::pair<dnnl::memory::format_tag, Shape> payload[] {
+            { dnnl::memory::format_tag::nChw16c,     staticShape },  // auto blocked
+            { dnnl::memory::format_tag::nhwc,        staticShape },  // permuted
+            { dnnl::memory::format_tag::nchw,        staticShape },  // plain
+            { dnnl::memory::format_tag::NChw16n16c,  staticShape },  // blocked for 2 dims
+            { dnnl::memory::format_tag::Acdb16a,     staticShape },  // same strides but not default order
+            { dnnl::memory::format_tag::nChw16c,     dynamicShape }, // auto blocked
+            { dnnl::memory::format_tag::nhwc,        dynamicShape }, // permuted
+            { dnnl::memory::format_tag::nchw,        dynamicShape }, // plain
+            { dnnl::memory::format_tag::NChw16n16c,  dynamicShape }, // blocked for 2 dims
+            { dnnl::memory::format_tag::Acdb16a,     dynamicShape }, // same strides but not default order
+    };
+
+    for (const auto &p : payload) {
+        redefineDesc(p.first, p.second);
+    }
+}
+
 TEST(MemDescTest, UndefinedStateConversion) {
```

Comment:
as i can see all test below can be covered by your test
so could you analyze tests below and move some test cases into your tests?
and remove tests below